### PR TITLE
Update Badge and Label

### PR DESCRIPTION
**Pull request by jonnystening**
## Updates
- Use new tokens
- Remove children destructure assignment (save lines)
- Rename information appearance as info (note: will be doing this to all components)
- Badge: add print styling

## Todo
- STG brand hero appearance uses `COLORS.text` for text color, other brands use white. This will be a thing throughout a number of components. The default/foreground colour mapping we used to have solved this. How do we proceed?

- Badge styling is not yet adjusted when the Badge is the child of a Button, this feature was in GUI2.0

### CODE REVIEW

File: components/badge/src/Badge.js [hunk 1]
```diff
@@ -8,38 +8,38 @@ import { jsx, useTheme } from '@westpac/core';
...
-			borderColor: appearance === 'faint' ? colors.border : colors[appearance].default,
+		'@media print': {
+			color: '#000',
+			backgroundColor: '#fff',
+			border: '1px solid #000',
 		},
 	};
 
-	return (
-		<span css={{ ...style.common, ...style.appearance }} {...props}>
```
Comment (jonnystening) [inline components/badge/src/Badge.js]: What do we reckon here?

Is there any benefit in separating (and then spreading) common styling and styling based on a prop (ie. appearance as in this case)?

File: components/badge/src/Badge.js [hunk 2]
```diff
@@ -8,38 +8,38 @@ import { jsx, useTheme } from '@westpac/core';
...
-		</span>
-	);
+	// Appearance styling
+	const styleAppearance = {
+		color: appearance === 'faint' ? COLORS.muted : '#fff',
+		backgroundColor: COLORS[appearance],
+		borderColor: appearance === 'faint' ? COLORS.border : COLORS[appearance],
+	};
+
+	return <span css={{ ...styleCommon, ...styleAppearance }} {...props} />;
```
Comment (jonnystening) [inline components/badge/src/Badge.js]: What do we reckon here?

Is there any benefit in separating (and then spreading) common styling and styling based on a prop (ie. `appearance` as in this case)?

Comment (jaortiz) [reply components/badge/src/Badge.js]: I would just inline them all. Dont think there is any benefit to separating them here. We were separating them in other components when we could make a mapping to the styles i.e.
```
const appearance  = {
   type1: {},
   type2: {}
}

return <div css={{...appearance[type]}} />
```

File: components/label/src/Label.js [hunk 3]
```diff
@@ -8,77 +8,62 @@ import { jsx, useTheme } from '@westpac/core';
...
-
-			'a&:hover, a&:focus': {
-				cursor: 'pointer',
-				backgroundColor: label.appearance[appearance].hover.backgroundColor,
-				borderColor: label.appearance[appearance].hover.borderColor,
-			},
+		'a&:hover, a&:focus': {
+			cursor: 'pointer',
+			backgroundColor: appearance === 'faint' ? '#fff' : COLORS.tints[`${appearance}50`],
+			borderColor: appearance !== 'faint' && COLORS.tints[`${appearance}50`],
```
Comment (jaortiz) [inline components/label/src/Label.js]: inline the styles. Same comment as badge

**Pull request merged**